### PR TITLE
About us page update

### PR DESCRIPTION
**Pull request by americano98**
This pull request brings design update for About us page.

Steps to check:

- open [preview](https://neon-next-git-about-us-page-upd-neondatabase.vercel.app/about-us)
- make sure that everything looks and works as expected

<details>
<summary>Design</summary>

about-us-1920-adaptive
</details>

### CODE REVIEW

**Comment by ruf-io** [comment]
@americano98 I switched the final number in the open source section and now it needs your design team's help to beautify 
image

**Comment by americano98** [comment]
@ruf-io thank you for the feedback, we have updated the styles for this section

**Comment by saimonkat** [comment]
Hi @ruf-io our designer insist to cut off "First Commits on Neon Repo" to "First Commits" on timeline's first step, if it's possible. We have a layout conflict on tablets. What do you think?

image

**Comment by saimonkat** [comment]
@ruf-io @danieltprice @rishi-raj-jain this PR is ready for merge, please somebody help with approve 🙏

File: src/components/pages/about/leadership/leadership.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      <Heading
+        className="max-w-[800px] text-[68px] font-medium leading-[0.9] tracking-extra-tight xl:max-w-[663px] xl:text-[56px] lg:max-w-[564px] lg:text-5xl md:text-[36px]"
+        tag="h2"
+        theme="black"
+      >
+        Leadership
+      </Heading>
+      <p className="mt-7 max-w-[740px] text-xl leading-snug tracking-extra-tight text-gray-new-50 xl:mt-5 xl:max-w-xl lg:mt-[18px] lg:max-w-lg lg:text-lg md:mt-3.5 md:text-base">
+        Neon was founded by a team of{' '}
+        <span className="text-white">Postgres hackers, systems and cloud engineers</span>.
```
Comment (danieltprice) [inline src/components/pages/about/leadership/leadership.jsx:63]:
```suggestion
        <span className="text-white">Postgres hackers, systems engineers, and cloud engineers</span>.
```

File: src/app/about-us/page.jsx [hunk 2]
```diff
@@ -1,33 +1,58 @@
...
+    <Layout>
+      <Hero />
+      <Timeline />
+      <ForDevelopers />
+      <OpenSource items={statistics} />
+      <Leadership />
+      <Investors />
+      <Cta
+        title="Become a part of our&nbsp;team"
+        description="We're looking for people who care deeply about quality to build it with us."
```
Comment (danieltprice) [inline src/app/about-us/page.jsx:49]:
```suggestion
        description="We're looking for people who care deeply about quality to build with us."
```

**Review by danieltprice — APPROVED** [review]

File: src/components/pages/about/leadership/leadership.jsx (hunk 1 above)
Comment (ruf-io) [reply src/components/pages/about/leadership/leadership.jsx:63]:
```suggestion
        <span className="text-white">database builders and Postgres hackers.</span>.
```

**Pull request merged**